pkgs/applications/misc/dump1090/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchgit, gnumake, pkgconfig, libusb, rtl-sdr }:
+
+stdenv.mkDerivation rec {
+  name = "dump1090";
```

Comment:
You should set a version like `"dump1090-<version>"`, so in this case a date for when the commit was made should be the version: `"dump1090-2014-11-01"`.